pkg/signup/service/signup_service.go

```diff
@@ -56,6 +56,11 @@ func NewSignupService(context servicecontext.ServiceContext, cfg ServiceConfigur
 // newUserSignup generates a new UserSignup resource with the specified username and userID.
 // This resource then can be used to create a new UserSignup in the host cluster or to update the existing one.
 func (s *ServiceImpl) newUserSignup(ctx *gin.Context) (*v1alpha1.UserSignup, error) {
+	username := ctx.GetString(context.UsernameKey)
+	if strings.HasSuffix(username, "crtadmin") {
```

Comment:
It's a minor thing, but I think that `-crtadmin` would be a more accurate suffix. 
Still, your code here should do the job and prevent us from signing up with our admin accounts.

Reply:
I would rather use the `crtadmin` and check it as a substring - not just as a suffix. If I'm not mistaken, you could create `mjobanek-crtadmin#` that would be then transformed by UserSignup controller into `mjobanek-crtadmin`. In other words, it's still a security issue.

Reply:
After our discussion on Slack, I believe that a username such as `mjobanek-crtadmin#` will not match any admin account since it will be transformed into `mjobanek-crtadmin-crt`, so I would stand with the `-crtadmin` suffix here. 
@MatousJobanek WDYT?